k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```diff
@@ -9,6 +9,13 @@ spec:
     controller:
       ingress:
         hostName: sds-build.platform.hmcts.net
+      secondaryingress:
```

Comment:
all of the below fields except the host name should be moved to the base file and the corresponding options removed in the sbox one as they are now common